site/src/scss/_homepage.scss

```diff
@@ -44,6 +44,27 @@
   }
 }
 
+%bubble-container {
```

Comment:
@jaymejohnson - Not sure how this snuck back in there.  Can you remove?

Reply:
I have no idea how that snuck back in. But it's gone now. Good catch!